### ISSUE TITLE
XLSX import range does not work for lines more than 2 and 3

### ISSUE THREAD

**Issue by agershun** [agershun/alasql#51]
```sql
    SELECT * FROM XLSX("k2015.xlsx",{sheetid:"Sheet1",range:"A1:CC2927"});
```
This works for range starting with A1 and A2, but not A3

**Comment by tolunoide** [comment]
Hi,
Same issue, I look into alasql.js code and found that the loop line 10678 (between row0 and row1) is not working. Looks like those var are not evaluate as an integer. I just put a "parseInt" on row0 and row1 at line 10658 and 10660 and it worked for me.

Best regards

**Comment by agershun** [comment]
Thank you!

**Comment by agershun** [comment]
Fixed in next version 0.0.43.
